packages/fxa-shared/subscriptions/types.ts

```diff
@@ -152,8 +152,10 @@ export type SentEmailParams = {
 };
 
 export const SubscriptionUpdateEligibility = {
+  CREATE: 'create',
   UPGRADE: 'upgrade',
   DOWNGRADE: 'downgrade',
+  BLOCKED_IAP: 'blocked_iap',
```

Comment:
These codes shouldn't be present on this enum. The upgrade eligibility util does not factor in IAP subscriptions or new (create).

Reply:
See both:

https://github.com/mozilla/fxa/blob/main/packages/fxa-shared/subscriptions/types.ts#L154-L156

and

https://github.com/mozilla/fxa/blob/main/packages/fxa-shared/subscriptions/types.ts#L163-L170